tests/src/Pdp/PublicSuffixListManagerTest.php

```diff
@@ -214,4 +214,20 @@ public function testGetProvidedListFromDefaultCacheDir()
         $this->assertTrue(array_key_exists('stuff-4-sale', $publicSuffixList['org']) !== false);
         $this->assertTrue(array_key_exists('net', $publicSuffixList['ac']) !== false);
     }
+
+    public function testGetDifferentPublicList()
```

Comment:
Please remove this test and add tests that demonstrate the different behavior between the ICANN domains and the private domains. This test doesn't really tell us much about how this new feature works and it doesn't prove that the new behavior functions as expected.